_layouts/website/page.html

```diff
@@ -0,0 +1,13 @@
+{% endblock %}
```

Comment:
I get an error when I try to build with that {% endblock %} at the beginning of the file. It seems to work correctly when I delete it.